db/python/layers/family.py

```diff
@@ -570,9 +304,9 @@ async def import_pedigree(
                             reported_sex=row.sex,
                         )
                     )
-                    external_participant_ids_map[
-                        row.individual_id
-                    ] = upserted_participant.id
+                    external_participant_ids_map[row.individual_id] = (
+                        upserted_participant.id
```

Comment:
I prefer the old formatting because I thought the upserted_participant.id value was wrapped in a tuple for a minute. NBD though.